config/vufind/SierraRest.ini

```diff
@@ -148,7 +164,7 @@ title_hold_bib_levels = a:b:m:d
 ;allowCancelingAvailableRequests = true
 
 ; This section allows modification of the default mapping from item status codes to
-; VuFind item statuses
+; VuFind item statuses. Defaults are
```

Comment:
This seems to trail off. Is something missing?

Reply:
Oops. That's when I realized that the defaults below were not quite complete. :) Now fixed.